src/bitmessagekivy/baseclass/popup.py

```diff
@@ -0,0 +1,233 @@
+# pylint: disable=import-error, attribute-defined-outside-init
+# pylint: disable=no-member, no-name-in-module, unused-argument, too-few-public-methods
+
+"""
+All the popup are managed here.
+
+"""
+
+from datetime import datetime
+
+from kivy.clock import Clock
+from kivy.metrics import dp
+from kivy.properties import StringProperty
+from kivy.uix.boxlayout import BoxLayout
+from kivy.uix.popup import Popup
+from kivy.app import App
+
+from bitmessagekivy import kivy_helper_search
+from bitmessagekivy.get_platform import platform
+
+from bitmessagekivy.baseclass.common import toast
+
+from addresses import decodeAddress
+from debug import logger
+
+
+class LoadingPopup(Popup):
+    """LoadingPopup class for kivy Ui"""
+
+    def __init__(self, **kwargs):
+        super(LoadingPopup, self).__init__(**kwargs)
+        Clock.schedule_once(self.dismiss_popup, 0.5)
+
+    def dismiss_popup(self, dt):
+        """Dismiss popups"""
+        self.dismiss()
+
+
+class GrashofPopup(BoxLayout):
```

Comment:
sorry but the class names are stupid and don't understand what they are supposed to mean